src/arch/genfortealloc.h

```diff
@@ -23,7 +26,15 @@ void forte_free(void *paData){
 
 inline
 void *forte_malloc(size_t paSize){
+#if defined(__GNUC__) && !defined(__clang__)
+  // BUG RESOLVED WONTFIX: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=85783
+  if (paSize <= PTRDIFF_MAX) {
```

Comment:
Does `std::PTRDIFF_MAX` not work, if yes please add `std::`, as assuming that std symbols are imported is not recommended according to StackOverflow

Reply:
Not work. Defined in header <cstdint> https://en.cppreference.com/w/cpp/types/climits

Reply:
Thanks for checking